config/crossSections13p6TeV.yaml

```diff
@@ -0,0 +1,729 @@
+  crossSec: 8.92
+  reference: XSDB QCD_PT-1000to1400_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:26
+  unc: 0.04204
+QCD_PT_1400to1800:
+  crossSec: 0.8103
+  reference: XSDB QCD_PT-1400to1800_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:30
+  unc: 0.003798
+QCD_PT_1800to2400:
+  crossSec: 0.1148
+  reference: XSDB QCD_PT-1800to2400_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:35
+  unc: 0.0005556
+QCD_PT_2400to3200:
+  crossSec: 0.007542
+  reference: XSDB QCD_PT-2400to3200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:36
+  unc: 3.779e-05
+QCD_PT_3200:
+  crossSec: 0.0002331
+  reference: XSDB QCD_PT-3200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:43
+  unc: 1.208e-06
+
+ST_s-channel_antitop:
+  crossSec: 1.43
+  reference: XSDB TbarBtoLminusNuB-s-channel-4FS_TuneCP5_13p6TeV modifiedOn 2024-04-03 19:51:58
+  unc: 0.0001633
+ST_s-channel_top:
+  crossSec: 2.278
+  reference: XSDB TBbartoLplusNuBbar-s-channel-4FS_TuneCP5_13p6TeV modifiedOn 2024-04-03 19:48:33
+  unc: 0.0003008
+ST_t-channel_antitop:
+  crossSec: '87.2'
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef
+  unc: +0.9 -0.8 (scale) +1.5 -1.3(PDF+alphaS) +1.8 -1.5(Total) +0.6 -0.7(mass) +0.2 -0.2 (Ebeam) +/-0.1 (Integration)
+ST_t-channel_top:
+  crossSec: '145.0'
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef
+  unc: +1.7 -1.1 (scale) +2.3 -1.5(PDF+alphaS) +2.8 -1.9(Total) +1.3 -0.9(mass) +0.4 -0.3 (Ebeam) +/-0.1 (Integration)
+ST_tW_antitop_2L2Nu:
+  crossSec: 87.9 * (0.5005) * (3 * 0.1086) * (0.3320)
+  reference: Value taken from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef, top/antitop % taken from XSDB, BRs taken from pdg
+  unc: 0.01296
+ST_tW_antitop_4Q:
+  crossSec: 87.9 * (0.5005) * (0.6741) * (0.665)
+  reference: Value taken from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef, top/antitop % taken from XSDB, BRs taken from pdg
+  unc: 0.01296
+ST_tW_antitop_LNu2Q:
+  crossSec: 87.9 * (0.5005) * ( ((3 * 0.1086) * (0.665)) + ((0.6741) * (0.3320)) )
+  reference: Value taken from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef, top/antitop % taken from XSDB, BRs taken from pdg
+  unc: 0.01296
+ST_tW_top_2L2Nu:
+  crossSec: 87.9 * (0.4995) * (3 * 0.1086) * (0.3320)
+  reference: Value taken from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef, top/antitop % taken from XSDB, BRs taken from pdg
+  unc: 0.01292
+ST_tW_top_4Q:
+  crossSec: 87.9 * (0.4995) * (0.6741) * (0.665)
+  reference: Value taken from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef, top/antitop % taken from XSDB, BRs taken from pdg
+  unc: 0.01292
+ST_tW_top_LNu2Q:
+  crossSec: 87.9 * (0.4995) * ( ((3 * 0.1086) * (0.665)) + ((0.6741) * (0.3320)) )
+  reference: Value taken from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef, top/antitop % taken from XSDB, BRs taken from pdg
+  unc: 0.01292
```

Comment:
I think these numbers were never replaced when changing reference form XSDB to calculation, updated with uncertainties from the twiki (which I think are % uncertainties so that should help???)